packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +101,38 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
+ *
+ * TransactionReceipt representation
+ * @property effectiveGasPrice - Effective gas price the transaction was charged at
+ * @property gasUsed - Gas used in the transaction
+ * @property l1Fee - Total used gas in hex
+ * @property status - The status of the transaction
+ */
+export interface TransactionReceipt {
```

Comment:
[There are more properties](https://docs.ethers.org/v5/api/providers/types/#providers-TransactionReceipt). But these are the only ones that's being used in client & backend for now.

Reply:
I guess less is more in this case :+1:  . One small thing should we take the opportunity to also include the rest of the properties in the docs?

Reply:
Addressed in https://github.com/MetaMask/core/pull/1592/commits/6355100ffa2a5b08ede47fc25589d1a06ef27869